docs/docs/spark-procedures.md

```diff
@@ -936,3 +936,40 @@ as an `UPDATE_AFTER` image, resulting in the following pre/post update images:
 |-----|--------|--------------|
 | 3   | Robert | UPDATE_BEFORE|
 | 3   | Dan    | UPDATE_AFTER |
+
+## Table stats
+
+### `compute_table_stats`
+
+This procedure calculates the [Number of Distinct Values (NDV) statistics](https://iceberg.apache.org/puffin-spec/#apache-datasketches-theta-v1-blob-type) for a specific table.
```

Comment:
sorry for the trouble, but can we make this a relative link so it works across ?

Reply:
the spec is found at [`format/puffin-spec.md`](https://github.com/apache/iceberg/blob/main/format/puffin-spec.md).
Do we want to reference a file outside the `docs` older?

Reply:
Yes can it happen?  The problem with this link is , as we version the docs, it will point to the latest version and not that version.